paasta_tools/contrib/rightsizer_soaconfigs_update.py

```diff
@@ -169,6 +169,34 @@ def get_recommendation_from_result(result, keys_to_apply):
     return rec
 
 
+def get_cassandra_recommendation_from_result(result, keys_to_apply):
+    rec = {}
+    for key in keys_to_apply:
+        val = result.get(key)
+        if not val or val == NULL:
+            continue
+        if key == "cpus":
+            rec["cpus"] = float(val)
+        elif key == "cpu_burst_percent":
+            rec["cpu_burst_percent"] = float(val)
+        elif key == "mem":
+            rec["mem"] = val
+        elif key == "disk":
+            rec["disk"] = val
+        elif key == "replicas":
+            rec["replicas"] = int(val)
+    return rec
+
+
+def get_recommendation_from_result(instance_type, result, keys_to_apply):
+    function_map = {
+        "cassandracluster": get_cassandra_recommendation_from_result,
+        "kubernetes": get_kubernetes_recommendation_from_result,
+        "marathon": get_kubernetes_recommendation_from_result,
```

Comment:
marathon no longer exists at yelp - we can obviate this entry from the mapping :)